python/lsst/ts/externalscripts/auxtel/latiss_acquire.py

```diff
@@ -0,0 +1,577 @@
+
+            dr_arcsec = np.sqrt(dx_arcsec**2 + dy_arcsec**2)
+
+            self.log.debug(
+                f"Calculated offsets [dx,dy] are [{dx_arcsec:0.2f}, {dy_arcsec:0.2f}] arcsec as calculated"
+                f" from sequence number {data_id['seq_num']} on Observation Day of {data_id['day_obs']}"
+            )
+
+            # Check if star is in place, if so then we're done
+            if dr_arcsec < self.target_pointing_tolerance:
+                self.log.info(
+                    "Acquisition completed successfully."
+                    f"Current radial pointing error of {dr_arcsec:0.2f} arcsec is within the tolerance "
+                    f"of {self.target_pointing_tolerance} arcsec. "
+                )
+                _success = True
+                break
+            else:
+                self.log.info(
+                    f"Current radial pointing error of {dr_arcsec:0.2f} arcsec exceeds the tolerance"
+                    f" of {self.target_pointing_tolerance} arcsec."
+                )
+
+                # Offset telescope, using only relative offsets
+                self.log.info("Applying x/y offset to telescope pointing.")
+
+                # Use persistent = False otherwise when we switch gratings
+                # it may keep an offset we no longer want
+                await self.atcs.offset_xy(
+                    x=dx_arcsec, y=dy_arcsec, relative=True, persistent=False
+                )
+
+            self.log.debug(
+                f"At end of iteration loop {iter_num+1}, success is {_success}."
+            )
+
+        # Check that maximum number of iterations for acquisition
+        # was not reached
+        if not _success:
+            self.log.debug(
+                "Failed to acquire star on target after "
+                f"{iter_num} images. Removing focus offset and "
+                "raising an exception"
+            )
+
+            raise RuntimeError(
+                f"Failed to acquire star on target after {iter_num} images."
+            )
+
+    async def latiss_slew_and_acquire(self):
+        # Setup instrument and telescope
+        # The following code sets up the instrument and slews the telescope
+        # simultaneously.
+
+        # If coordinates are provided then use those and set the target name.
+        if self.object_ra and self.object_dec:
+            self.log.debug("Using slew_icrs (object coordinate designation)")
+            _slew_coro = self.atcs.slew_icrs(
+                self.object_ra,
+                self.object_dec,
+                target_name=self.object_name,
+                rot=self.rot_value,
+                rot_type=getattr(RotType, self.rot_type),
+                slew_timeout=240,
+                az_wrap_strategy=self.azimuth_wrap_strategy,
+                time_on_target=self.time_on_target,
+            )
+        # Otherwise use the target name to query Simbad for coordinates.
```

Comment:
I think you can remove this comment and put this information in the method docstring.